File: examples/ticket.py
import json
import sys
from pathlib import Path

from openjev import OpenJevClient

request = json.loads(Path(__file__).with_suffix(".json").read_text())
client = OpenJevClient(sys.argv[1] if len(sys.argv) > 1 else "http://127.0.0.1:8000")
print(json.dumps(client.system_one(**request), indent=2))

File: examples/ticket.mjs
import { readFile } from "node:fs/promises";
import { OpenJevClient } from "../sdk/js/index.js";

const request = JSON.parse(await readFile(new URL("./ticket.json", import.meta.url), "utf8"));
const client = new OpenJevClient({ baseUrl: process.argv[2] ?? "http://127.0.0.1:8000" });
console.log(JSON.stringify(await client.systemOne(request), null, 2));
